### ISSUE TITLE
Comparison involving remote workspace folder fails

### ISSUE THREAD

**Issue by mforkel** [L13/vscode-diff#78]
I tried to use Diff Folders in a workspace that contains local folders and remote folders accessed with the [SSH FS extension](https://github.com/SchoofsKelvin/vscode-sshfs). An attempt to compare a local folder to a remote folder results in an error because `/` is scanned instead of the remote folder (VS Code 1.54.1 with Remote - WSL 0.54.3):
```
[22:40:38] Start comparing "/path/of/local/folder" ↔ "/"
[22:40:38] Scanning "/path/of/local/folder"
[22:40:38] Found 44 entries
[22:40:38] Scanning "/"
[22:40:38] Error: EACCES: permission denied, scandir '/lost+found'
Error: EACCES: permission denied, scandir '/lost+found'
```
Comparing a file in a local folder to a file in a remote folder works fine.

The [SSH FS documentation for remote workspace folders](https://github.com/SchoofsKelvin/vscode-sshfs#remote-workspace-folders) claims that SSH FS works with extensions using the [`vscode.workspace.fs` API](https://code.visualstudio.com/updates/v1_37#_vscodeworkspacefs). May be use of that API could solve the reported issue?

**Comment by L13** [comment]
Hi, comparing remote files with local files is not possible right now because the vscode api does not provide all features which nodejs does. When I started this extension, where was no vscode api for the file system. Later I tried to switch to the vscode api, but I came up that it doesn't work. So there are no plans to do this. Sorry for the bad news.

**Comment by mforkel** [comment]
Thanks for letting me know! And for providing this extension in the first place. Could you identify the enhancements to the API that would be required by your (and probably other) extensions? It seems to me the VS Code people are open for suggestions :-)

**Comment by L13** [comment]
Implementing the vscode api also means a lot of work. We talk about weeks. This extension is a spare time project. Spending so much time for something what I don't need right now is not what I prefer. ;) I have written the extension because I was not really happy with the alternatives. So I decided to write my own one. The reason why I published the extension was "maybe, someone needs it too", but as it is.
